processes/protocol-upgrade.md

```diff
@@ -33,7 +33,7 @@ Ethereum has regular protocol upgrades that introduce performance, security and
 
 ### Mainnet
 
-- Mainnet should not have an upgrade date set until all testnets have been upgraded.
+- Mainnet should not have an upgrade date set until at least one testnet has been upgraded.
```

Comment:
I think this is reasonable. My 2c from a client dev perspective:

Things can always go wrong - we could set hard requirements and be more conservative (e.g. wait at least a week after all testnets have upgraded) and delay shipping even if the failure rate is extremely low; But I see more benefit in being optimistic and focusing on improving our testing process to reduce the failure rates. (which I believe we’re constantly getting better at!)

In my opinion, doing more devnet iterations with clear goals until things are robust is much more meaningful than having a hard requirement to wait for all testnets to upgrade.

Reply:
This isn't meant to decide when a client puts the dates into their releases. I don't think we should decide that for the client teams here.

Reply:
I'm against this change.  We can (and in Fusaka's case have) propose a target date. but it shouldn't be confirmed until all testnets have been upgraded.

Though even with a proposed date, if there is an issue which requires more time, then we should be prepared that the wider world will see this as core devs delaying an upgrade.